typings.d.ts

```diff
@@ -115,11 +110,15 @@ declare module 'contentful-ui-extensions-sdk' {
     /** Removes the value for the field and locale. */
     removeValue: (locale?: string) => Promise<void>
     /** Calls the callback every time the value of the field is changed by an external event or when setValue() is called. */
-    onValueChanged: onValueChangeWithLocaleType
-    onValueChanged: onValueChangedType
+    onValueChanged: {
+      (callback: (value: any) => void): Function
```

Comment:
Is using `Function` ok? Should we simply instead do `() => void` and avoid using the Primitive?

Reply:
yeah `() => void` should be good